### DISCUSSION TITLE
Would you like a sequel to this parts addon named FTPxl?

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/FuelTanksPlus#133] [Polls]
I am considering after I finish updating this glorious part addon of creating a sequel named:

# Fuel Tanks Plus Xtra Large

* size4 (5m)
* size5 (7.5m)
* size6 (10m)

**Comment by Nico314159** [comment]
Couldn't those just be part of SpaceY?

**Reply by zer0Kerbal** [reply]
As in all in one package? or under the SpaceY `umbrella`?

Reason don't want to put these into the existing package is choice. I plan on doing it so players can choose which sizes they want, and if I do it right - the textures / models can be used for multiple parts - saving allot of space and texture memory.
